src/pheval_exomiser/post_process/post_process_results_format.py

```diff
@@ -97,25 +97,53 @@ def _find_relevant_score(self, result_entry) -> float:
         """Return score from Exomiser result entry."""
         return round(result_entry[self.score_name], 4)
 
-    def extract_pheval_variant_requirements(self) -> [PhEvalVariantResult]:
+    def _filter_for_acmg_assignments(
+        self, variant: PhEvalVariantResult, score: float, variant_acmg_assignments: dict
+    ) -> bool:
+        """Filter variants if they meet the PATHOGENIC or LIKELY_PATHOGENIC ACMG classification."""
+        for assignment in variant_acmg_assignments:
+            if variant == PhEvalVariantResult(
+                chromosome=self._find_chromosome(assignment["variantEvaluation"]),
+                start=self._find_start_pos(assignment["variantEvaluation"]),
+                end=self._find_end_pos(assignment["variantEvaluation"]),
+                ref=self._find_ref(assignment["variantEvaluation"]),
+                alt=self._find_alt(assignment["variantEvaluation"]),
+                score=score,
+            ):
+                if (
+                    assignment["acmgClassification"] == "PATHOGENIC"
+                    or assignment["acmgClassification"] == "LIKELY_PATHOGENIC"
```

Comment:
No need to heed, but I tend to think static vars are better than STRINGS, also for debugging and finding stuff in code.

Reply:
Yes, I agree!